pkl-core/src/test/kotlin/org/pkl/core/LanguageSnippetTestsEngine.kt

```diff
@@ -58,6 +58,15 @@ abstract class AbstractLanguageSnippetTestsEngine : InputOutputTestEngine() {
     return expectedOutputDir.resolve(stdoutPath)
   }
 
+  override fun beforeAll() {
+    // disable SHA verification for packages
+    IoUtils.setTestMode()
```

Comment:
I'd love to get rid of this.

Reply:
The snippet tests now pass without setting `IoUtils.setTestMode()`.

I'm no longer sure what the "Qualify of life improvement" in `PackageResolvers.verifyPackageMetadataBytes` accomplishes. It checks for `IoUtils.isTestMode()`, which I think is always `false` unless when running native tests.

Reply:
Our snippet tests used to use a real server, but we switched to a pre-seeded cache because our tests were really flakey.